WooCommerce/src/main/kotlin/com/woocommerce/android/ui/feedback/SurveyType.kt

```diff
@@ -17,5 +18,7 @@ enum class SurveyType(private val untaggedUrl: String, private val milestone: In
             else -> ""
         }
 
+    private val appVersion = "&app-version=${BuildConfig.VERSION_NAME}"
```

Comment:
It might be better to rename this as `appVersionTag` so we're consistent with the other `*Tag` properties. 

```suggestion
    private val appVersionTag = "&app-version=${BuildConfig.VERSION_NAME}"
```